include/aktualizr-lite/cli/cli.h

```diff
@@ -34,12 +34,30 @@ enum class StatusCode {
   InstallRollbackFailed = 130,
 };
 
+/**
+ * The installation mode to be applied. Specified during InstallContext context initialization.
+ */
+enum class PullMode {
+  /**
+   * Default mode, do pull target during install operation.
+   */
+  PerformPull = 0,
+  /**
+   * Do no pull target during install. Target is expected to be pulled before.
+   */
+  SkipPull,
+};
+
 StatusCode CheckIn(AkliteClient &client, const LocalUpdateSource *local_update_source);
 
+StatusCode Pull(AkliteClient &client, int version = -1, const std::string &target_name = "",
+                InstallMode install_mode = InstallMode::All, bool force_downgrade = true,
+                const LocalUpdateSource *local_update_source = nullptr);
 
 StatusCode Install(AkliteClient &client, int version = -1, const std::string &target_name = "",
                    InstallMode install_mode = InstallMode::All, bool force_downgrade = true,
```

Comment:
Not related to this PR. This `force_downgrade` looks a bit off now. I think it can be replaced with another install mode. Let's think about it and make corresponding changes in another PR.